components/python/scandir/scandir-27.p5m

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
You can still do this with PYVER, just set PYTHON_VERSIONS in Makefile. But this allows in future easily add one more python version.